.github/workflows/gh-ci.yaml

```diff
@@ -212,8 +212,16 @@ jobs:
     - name: deploy_docs
       if: github.event_name != 'pull_request'
       run: |
-        # place the deploy call here
-        echo "Oh, maple syrup roast parsnips [Richard Gowers]"
+        python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
+        cd package/MDAnalysis
+        export VERSION=$(python -c "import version; print(version.__version__)")
+        cd -
+        export GH_DOC_BRANCH=$GITHUB_REF
```

Comment:
Every env var (except the oauth2 token) could also become a commandline argument for the `deploy_docs.sh` script. However, as long as it's working the way it is I don't see much need to change it.

Reply:
I don't like those because I can never remember which order things are in 😅 Regardless it's probably better than cluttering up the environment. I think I will actually take @IAlibay's suggestion and move things to env / move the whole deployment within `gh-ci.yaml`, unless there are objections; the general pros for having a separate script (modularity, being able to deploy docs outside of CI) are somewhat negated by the heavy dependence on environment variables set in the CI.